build-tests/x86/archlinux/test-image-live-disk-kis/appliance.kiwi

```diff
@@ -31,23 +31,23 @@
         <rpm-check-signatures>false</rpm-check-signatures>
     </preferences>
     <preferences profiles="Live">
-        <type image="iso" flags="overlay" hybridpersistent_filesystem="ext4" hybridpersistent="true" firmware="efi"/>
+        <type image="iso" flags="overlay" hybridpersistent_filesystem="ext4" hybridpersistent="true" firmware="efi" kernelcmdline="console=ttyS0"/>
```

Comment:
same here, I'd say the bootloader entry here makes sense too.

Reply:
done